db/db_impl.cc

```diff
@@ -5096,8 +5052,6 @@ void DBImpl::MaybeFlushColumnFamilies() {
     alive_log_files_.begin()->getting_flushed = true;
   }
 
-  WriteContext context;
```

Comment:
The purpose of WriteContext is to store pointers to superversions_to_free and memtables_to_free in order to deallocate them after release mutex lock. Seems during previous code changes the WriteContext is deallocate with mutex locked, and I'm fixing it.